### DISCUSSION TITLE
How can i pull or introspect a defined list tables

### DISCUSSION THREAD

**Discussion by kaykhan** [prisma/prisma#15775] [Q&A]
HI,

I am working on a service and i would like to only introspect or pull a handful of tables rather than the entire database tables. Is there some way i can do that with the following command `npx prisma db pull`

**Comment by nurul3101** [comment]
Hi @kaykhan 👋 
It's not possible at the moment, `npx prisma db pull` will introspect all the database tables. You would need to manually delete the not required tables from the schema file after introspection. 

<img width="724" alt="Screenshot 2022-10-13 at 7 02 16 PM" src="https://user-images.githubusercontent.com/13049676/195610802-5d4a0aba-8b64-4ff5-a68a-e49f7662a581.png">

Reference: https://www.prisma.io/docs/concepts/components/introspection#introspecting-only-a-subset-of-your-database-schema

**Reply by vladi-stevanovic** [reply]
👋 Hi @kaykhan did you perhaps have a chance to check Nurul's recommendation? Let us know if you still have any questions, or feel free to "Mark as answer" if this discussion is now solved!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
